### PR TITLE
#181339485 signup page

### PR DESCRIPTION
**Pull request by Eli250**
#### What does this PR do?

Signup page

#### Description of Task to be completed?

To be able to use the application and access authenticated routes, Users should be able to Signup and or Register using Sign up forms on the signup route.

#### How should this be manually tested?

Locally switch to the branch `feature/181339485-signup-page` and open the register link.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[181339485](https://www.pivotaltracker.com/n/projects/2555280)

#### Screenshots (if appropriate)

Screen Shot 2022-05-13 at 21 32 03

Screen Shot 2022-05-13 at 21 28 17

#### Questions:

### CODE REVIEW

File: src/redux/actions/signup.action.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  SIGNUP_SUCCESSFUL,
+} from '../types/signup.types';
+// import axios from 'axios';
+
+// export const SignupAction = (userData) => async (dispatch) => {
+//   try {
+//     dispatch({
+//       type: SIGNUP,
+//       payload: 'loading',
+//     });
```
Comment (Code-250) [inline src/redux/actions/signup.action.js:14]: remove commented codes

**Review by Code-250 — COMMENTED** [review]
please remove code that you have commented

File: src/redux/actions/signup.action.js (hunk 1 above)
Comment (Eli250) [reply src/redux/actions/signup.action.js]: Done

**Review by Code-250 — APPROVED** [review]

**Review by Gabin-ishimwe — APPROVED** [review]

**Review by niyonkurueric — APPROVED** [review]

**Comment by ncutixavier** [comment]
### Comments

- Your validation is not yet efficient
image

- You need to reset your form after successful register

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**